### PR TITLE
defaultMode field on secret causing oadm_router to detect changes

### PR DESCRIPTION
**Pull request by joelddiaz**
when there were none

ignore defaultMode when trying to determine if there has been a configuration change

### CODE REVIEW

File: ansible/roles/lib_openshift_3.2/library/oc_obj.py [hunk 1]
```diff
@@ -934,7 +934,7 @@ def create(self, files=None, content=None):
         '''
            Create a config
 
-           NOTE: This creates the first file OR The first conent.
+           NOTE: This creates the first file OR the first conent.
```
Comment (kwoodson) [inline ansible/roles/lib_openshift_3.2/library/oc_obj.py]: This change is only in the library file.  Needs to move to source?

**Review by kwoodson — APPROVED** [review]

**Comment by tiwillia** [comment]
[test]

**Comment by tiwillia** [comment]
If all the stars are properly aligned, this should pass [test] now, or at least stop failing them because of the ruamel.yaml import.

**Comment by tiwillia** [comment]
@joelddiaz the test failures appear to be actual failures now, although they look to have existed prior to your change. Sorry about the issues.

File: ansible/roles/lib_openshift_3.2/library/oc_obj.py (hunk 1 above)
Comment (detiber) [inline ansible/roles/lib_openshift_3.2/library/oc_obj.py:937]: s/conent/content

**Review by kwoodson — APPROVED** [review]

**Pull request merged**